pkg/tracer/tracer_linux_test.go

```diff
@@ -475,6 +479,86 @@ func TestUDPSendAndReceive(t *testing.T) {
 	tr.Stop()
 }
 
+func TestHTTPRequestLog(t *testing.T) {
+	tr, err := NewTracer(MakeTestConfig())
+	assert.NoError(t, err)
+	assert.NoError(t, tr.Start())
+
+	testServer := createTestHTTPServer()
+
+	// perform test calls to HTTP server that should be caught by BPF the tracer
+	statusCode, respText := runGETRequest(t, testServer, "/")
+	assert.Equal(t, 200, statusCode)
+	assert.Equal(t, "OK", respText)
+	// give it a little time to settle in buffers
+	time.Sleep(200 * time.Millisecond)
```

Comment:
Is there a different way we can do this? This usually leads to erratic tests.

Reply:
We check "GetConnections()" function and we don't have status if it's ready with new data. Perhaps we can retry until some timeout, like "eventually {}" in out scala tests

Reply:
Made it with eventually check with bigger timeout but smaller intervals to re-check